runtime/src/iree/vm/stack.c

```diff
@@ -12,6 +12,7 @@
 #include <string.h>
 
 #include "iree/base/api.h"
+#include "iree/base/assert.h"
```

Comment:
Do not include iree/base/assert.h outside of iree/base/ - iree/base/api.h includes it already